packages/react/src/misc/useProfilePrice.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryResult } from '@apollo/client';
+import { UnspecifiedError } from '@lens-protocol/api-bindings';
+import { useEffect, useState } from 'react';
+
+import { ReadResult, useReadResult } from '../helpers/reads';
+import {
+  ProfilePriceResult,
+  useProfilePriceController,
+} from './adapters/useProfilePriceController';
+
+export type { ProfilePriceResult };
+
+/**
+ * Retrieve the prices for minting a new Lens Profile, denominated in various currencies.
+ *
+ * @example
+ * ```tsx
+ * const { data, error, loading } = useProfilePrice();
+ * ```
+ *
+ * @category Misc
+ * @group Hooks
+ */
+export function useProfilePrice(): ReadResult<ProfilePriceResult> {
```

Comment:
Should we consolidate the use of plural/singular? Sometimes we say `ProfilePrices`, here it's singular.

Reply:
I made it plural as we expect to add more currencies soon